src/ui-test/tests/codelenses/jbang.test.ts

```diff
@@ -0,0 +1,145 @@
+import * as path from 'path';
+import { variables } from '../../variables';
+import {
+    after,
+    before,
+    CodeLens,
+    DebugToolbar,
+    EditorTabNotFound,
+    EditorView,
+    error,
+    errors,
+    repeat,
+    Repeat,
+    resources,
+    TextEditor,
+    until,
+    VSBrowser,
+    workspaces
+} from 'vscode-uitests-tooling';
+
+const file = 'JBang.yaml';
+
+describe('JBang codelens debug', function () {
+    this.timeout(45000);
+    this.slow(10000);
+    const workspace = workspaces.createWorkspace(VSBrowser.instance, variables.launcher.WORKBENCH_DIR);
+    const manager = resources.createResourceManager(VSBrowser.instance, workspace, variables.launcher.RESOURCES_DIR);
+
+    before(async function () {
+        await workspace.delete();
+        await workspace.create();
+        await manager.copy(file);
+        await workspace.open();
+    });
+
+    after(async function () {
+        await clearBreakpoint(9, 5000).catch(console.error);
+        await clearBreakpoint(15, 5000).catch(console.error);
+        await workspace.close(5000).catch(console.error);
+        await workspace.delete();
+    });
+
+    it('JBang.yaml codelen debug works', async function () {
+        await openFile(path.join(workspace.path, file));
+        await addBreakpoint(9);
+        await addBreakpoint(15);
+
+        // There is a case when codelen becomes stale and must
```

Comment:
```suggestion
        // There is a case when codelens becomes stale and must
```

a codelens, 2 codelenses.